temp/ribbon_testing/data/3S_genes.bed

```diff
@@ -0,0 +1,2239 @@
+SN15_SOL_Chromosome_05	4558	5204	.	.	-
```

Comment:
Why in `temp` directory? Perhaps just move these `ribbon_testing` to regular testing if they pass?

Reply:
oops, local test. Shouldn't be here.